Create Executive Vote - June 16, 2023.md (#2318)
makerdao/community: governance/votes/Executive Vote - June 16, 2023.md

---
title: Template - [Executive Vote] Out-of-Schedule Executive Vote - RWA014-A (Coinbase Custody) DAO Resolution - June 16, 2023

> summary: Approving a DAO Resolution pertaining to RWA014-A (Coinbase Custody).

Reviewer: Summary might as well say what it's intended to achieve? 'To help ensure sufficient DAI liquidity' or something.

Author: updated

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
- A DAO Resolution pertaining to RWA014-A (Coinbase Custody) will be approved, as detailed below.

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

### RWA014-A (Coinbase Custody) DAO Resolution

> As per this [forum post](INSERT LINK HERE from the Stability Scope Responsible Facilitators, detailing the rationale for this out-of-schedule executive proposal under [MIP24](https://mips.makerdao.com/mips/details/MIP24), approval of this executive proposal will signal Maker Governance's agreement to the following DAO Resolution to James Asset Trust 3, if this executive proposal passes.

Reviewer: Fairly sure you'll pick it up, but you are also missing a closing parenthesis here, in addition to the link.

The DAO Resolution may be seen on IPFS [here](https://gateway.pinata.cloud/ipfs/QmYPbYoPjrSzBu5gt9tip78siG1gFvY8K4HTkHEFJgMmM8).

> The IPFS hash will be added to the code of the spell as a constant in order to provide added clarity of the on chain approval.

Reviewer: ```suggestion
The IPFS hash has been added to the executive spell code as a constant in order to provide added clarity of the on chain approval.
```

> The raw text is included below for clarity:

Reviewer: ```suggestion
The raw text of the resolution is provided below for clarity:
```
